### DISCUSSION TITLE
4k vs 1080 instances / Search management vs download management

### DISCUSSION THREAD

**Discussion by PhattestMattest** [Feramance/qBitrr#106] [Q&A] [answered]
Up top id like to apologize for my incompetence in this area. Iv always down my downloading through nzb and only recent ventured into torrenting. 

First question, I have a single instance of radarr that searches for 4k and 1080p media based off radarr's profile and uses prowlerr to do the searching. Do I have to set the config for both sections or do I use the 4K_instance setting under the [radarr-1080] section of the config.

Second question, since prowler is doing my searching, can I set the search.entity setting to false and just fill in the info for the arrs to have qBitrr detect stalled / < 1.00 availability / no seeder torrents? 

Thanks in advanced for any help!

**Comment by Feramance** [comment, marked as answer]
Hi, no worries we've all been there at some point.

First answer, no. The config is an example of what you could do, you only need to configure as many Radarr instances as you have, same with Sonarr. So 1 instance, then only configure 1, you can safely delete the others and theoretically qBitrr can support an indefinite number of instances as it runs a process per instance.

Second answer, you can though I think you misunderstood. The search setting is whether you want qBitrr to send a request to Radarr/Sonarr to force search (they use Prowlarr). qBitrr itself doesn't search, it tells the arr apps to search.

Hope this clears things up
